test/extensions/filters/network/redis_proxy/config_test.cc

```diff
@@ -206,6 +206,39 @@ stat_prefix: foo
   cb(connection);
 }
 
+TEST(RedisProxyFilterConfigFactoryTest, RedisProxyModulesProto) {
```

Comment:
Do you have any tests that actually exercise that custom commands work? Can you potentially add an integration test of some type?